kubernetes/ingress/zooniverse.org.yaml

```
@@ -230,6 +230,15 @@ spec:
           serviceName: http-frontend
           servicePort: 80
         path: /(.*)
+
+  - host: mapping-viz-staging.zooniverse.org
+    http:
+      paths:
+      - backend:
+          serviceName: tove-production-app
```

Not sure what the correct `serviceName` should be -- will you be setting that as part of adding a deployment Action on the mapping-viz-tools repo?

```suggestion
          serviceName: mapping-viz-tools-staging
```

---

Copy paste miss. The service doesn't exist yet, it will be added to that repo as a k8s template. Thanks.